mu-plugins/blocks/global-header-footer/footer.php

```diff
@@ -115,7 +115,7 @@
 
 		<?php else : ?>
 			<!-- Use text so it can be translated. -->
-			<span class="global-footer__code_is_poetry">
+			<span class="global-footer__code_is_poetry <?php echo str_ends_with( $color_scheme , 'white' ) ? 'is-dark' : '' ;?>">
```

Comment:
Some lint issues with the spacing here:
```suggestion
			<span class="global-footer__code_is_poetry <?php echo str_ends_with( $color_scheme, 'white' ) ? 'is-dark' : ''; ?>">
```

Reply:
I'll run the linter and open a pr so we can add it to the workflow.